modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/ItConfigCommandTest.java

```diff
@@ -88,7 +90,8 @@ public void setAndGetWithManualHost() {
         assertEquals(0, exitCode);
         assertThat(
                 out.toString(UTF_8),
-                containsString("Node configuration was updated successfully")
+                containsString("Node configuration updated. "
+                        + fg(Color.RED).mark("Restart the node to apply changes."))
```

Comment:
I think the red color indicates an error but this is not the one. I would say that the yellow color or bold text would be enough.